cypress/integration/tests/applicationinventory/manageimports/pagination.test.ts

```diff
@@ -0,0 +1,240 @@
+/// <reference types="cypress" />
+
+import {
+    login,
+    clickByText,
+    selectItemsPerPage,
+    click,
+    importApplication,
+    openManageImportsPage,
+} from "../../../../utils/utils";
+import { navMenu } from "../../../views/menu.view";
+import { applicationinventory, button, tdTag, trTag, deleteAction } from "../../../types/constants";
+import { actionButton } from "../../../views/applicationinventory.view";
+import { ApplicationInventory } from "../../../models/applicationinventory/applicationinventory";
+
+import * as commonView from "../../../views/common.view";
+import { BusinessServices } from "../../../models/businessservices";
+
+const businessService = new BusinessServices("Finance and HR");
+const filePath = "app_import/";
+var applicationsList: Array<ApplicationInventory> = [];
+const filesToImport = [
+    "valid_application_rows.csv",
+    "mandatory_and_empty_rows.csv",
+    "non_existing_tags_business_service_rows.csv",
+];
+
+describe("Manage imports pagination validations", function () {
+    before("Login and Create Test Data", function () {
+        // Perform login
+        login();
+
+        // Create business service
+        businessService.create();
+
+        // Navigate to Application inventory tab
+        clickByText(navMenu, applicationinventory);
+        cy.wait(5000);
+        var rowsToCreate = 0;
+
+        // Import multiple csv files
+        function importMultipleFiles(num): void {
+            for (let i = 0; i < rowsToCreate; i++) {
+                importApplication(filePath + filesToImport[i]);
+            }
+        }
+
+        // Get the current table row count and create appropriate test data rows
+        openManageImportsPage();
+        selectItemsPerPage(100);
+        cy.get(commonView.appTable)
+            .next()
+            .then(($div) => {
+                if (!$div.hasClass("pf-c-empty-state")) {
+                    cy.get(".pf-c-table > tbody > tr")
+                        .not(".pf-c-table__expandable-row")
+                        .find("td[data-label='File name']")
+                        .then(($rows) => {
+                            var rowCount = $rows.length;
+                            if (rowCount <= 10) {
+                                rowsToCreate = 11 - rowCount;
+                            }
+                            if (rowsToCreate > 0) {
+                                importMultipleFiles(rowsToCreate);
+                            }
+                        });
+                } else {
+                    rowsToCreate = 11;
+                    importMultipleFiles(rowsToCreate);
+                }
+            });
+
+        // Create objects for imported apps
+        const appsImported = ["Import-app-1", "Import-app-2", "Import-app-5", "Import-app-6"];
```

Comment:
Why are we hardcoding this ? 
What if any of these are not deleted?

Reply:
Because these app names are present in the imported csv. There is no way to know what application name was imported. And for deletion we have logic to check if these apps got actually created, then only, they will get deleted.